lib/logmsg/logmsg.h

```diff
@@ -142,6 +142,34 @@ enum
   __UNUSED_LF_LEGACY_MSGHDR    = 0x00020000,
 };
 
+typedef NVType LogMessageValueType;
```

Comment:
NVType is also typedefed in nvtable.h to a different type: guint8.
Typedef redefinitions produce compiler errors even without the `-pedantic` flag in older gcc versions, so I think the redef should be eliminated.

Reply:
It is not NVType that gets defined here, it is LogMessageValueType that becomes an alias for NVType.

I choose this, so that nvtable does not need to care about
the specifics and semantic meaning of the type field. 
LogMessage should be responsible for translation.

Reply:
Sorry again. I don't know what I thought originally when I wrote this...